topostats/tracing/dnatracing.py

```diff
@@ -792,34 +792,6 @@ def measure_contour_length(self):
                         del hypotenuse_array
                         break
 
-    def writeContourLengths(self, filename, channel_name):
-        if not self.contour_lengths:
-            self.measure_contour_length()
-
-        with open(f"{filename}_{channel_name}_contours.txt", "w") as writing_file:
-            writing_file.write("#units: nm\n")
-            for dna_num in sorted(self.contour_lengths.keys()):
-                writing_file.write("%f \n" % self.contour_lengths[dna_num])
-
-    # FIXME : This method doesn't appear to be used here nor within pygwytracing, can it be removed?
-    def writeCoordinates(self, dna_num):
-        # FIXME: Replace with Path()
-        if not os.path.exists(os.path.join(os.path.dirname(self.filename), "Coordinates")):
-            os.mkdir(os.path.join(os.path.dirname(self.filename), "Coordinates"))
-        directory = os.path.join(os.path.dirname(self.filename), "Coordinates")
-        savename = os.path.join(directory, os.path.basename(self.filename)[:-4])
-        for i, (x, y) in enumerate(self.splined_traces[dna_num]):
-            try:
-                coordinates_array = np.append(coordinates_array, np.array([[x, y]]), axis=0)
-            except NameError:
-                coordinates_array = np.array([[x, y]])
-
-        coordinates = pd.DataFrame(coordinates_array)
-        coordinates.to_csv(f"{savename}_{dna_num}.csv")
-
-        plt.plot(coordinates_array[:, 0], coordinates_array[:, 1], "ko")
-        plt.savefig(f"{savename}_{dna_num}_coordinates.png")
-
```

Comment:
While it's not used in the main branch, I do use it in my branch. When I merged some of my work into the main back when we were on TopoStats 1.0, I did not enable it as some of my work wasn't complete, but I did intend to keep it. The idea of this method is to produce the coordinates of the traces, so that when other tracing based measurements went wrong, I could always look at those to check whether the tracing was satisfatory, so as long as the functionalities are covered elsewhere,  deleting them here should be fine.